ZConfig/components/logger/handlers.xml

```diff
@@ -12,46 +12,126 @@
     </description>
     <key name="formatter" datatype="dotted-name" required="no">
       <description>
-        Logging formatter class.  The default is 'logging.Formatter'.
+        Logging formatter class.  The default is :class:`logging.Formatter`.
         An alternative is 'zope.exceptions.log.Formatter',
         which enhances exception tracebacks with information from
-        __traceback_info__ and __traceback_supplement__ variables.
+        ``__traceback_info__`` and ``__traceback_supplement__``
+        variables from each stack frame.
       </description>
     </key>
     <key name="dateformat"
-         default="%Y-%m-%dT%H:%M:%S"/>
+         default="%Y-%m-%dT%H:%M:%S">
+      <description>
+        Timestamp format used for the 'asctime' field.
+
+        This is used with Python's :func:`time.strftime` function,
+        so must be compatible with that function on the host platform.
+      </description>
+    </key>
     <key name="level"
          default="notset"
-         datatype="ZConfig.components.logger.datatypes.logging_level"/>
+         datatype="ZConfig.components.logger.datatypes.logging_level">
+      <description format="rest">
```

Comment:
The `format` attribute seems out of place (I don't see it anywhere else).

Reply:
Indeed, it appears to be unnecessary.

Changed in commit 5f85187.